### DISCUSSION TITLE
How can I use a variable in Place of the Table Name?

### DISCUSSION THREAD

**Discussion by darrylmorley** [prisma/prisma#9284] [Q&A] [answered]
Hi all,

How can I use a variable in place of the table name when creating a record, i.e.:
```
prisma.tableNameVariableHere.create({})
```

I'm trying to use the table from this object:
```
meta: {
    site: 'somesite.co.uk',
    table: 'log',
    level: 'info',
    message: 'i wrote this for you',
    [Symbol(level)]: 'info',
    [Symbol(splat)]: [ [Object] ],
    [Symbol(message)]: '{"site":"somesite.co.uk","table":"log","level":"info","message":"i wrot ethis for you"}'
  }
```

**Comment by darrylmorley** [comment, marked as answer]
Stupid moment, please diregard.

**Reply by ThomasPayWut** [reply]
I am apparently having the same stupid moment because I can't figure this out.

**Reply by ThomasPayWut** [reply]
Okay, never mind. haha, I figured it out  --> prisma[tableName].create({})

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
